test/no-reinstall.js

```diff
@@ -1,6 +1,8 @@
 describe('when files are installed', function () {
     it('should not reinstall them', function (done) {
 
+        this.timeout(120000);
```

Comment:
Why did you have to add this line while previously it was not present? Thanks

Reply:
Unfortunately, the CDN which serves the chromium edge driver takes a bit longer to download the zip sometimes, so I had to increase the test timeout in order to make it always pass. Usually it should take a few seconds only but I wanted to make this change so future changes will also make all tests pass.

Reply:
👍